compiler/common-artifacts/src/TestDataGenerator.cpp

```diff
@@ -45,6 +45,9 @@ H5::PredType hdf5_dtype_cast(const loco::DataType loco_dtype)
   {
     case loco::DataType::U8:
       return H5::PredType::NATIVE_UINT8;
+    case loco::DataType::S4:
+      // NOTE there is no INT4
+      return H5::PredType::NATIVE_INT8;
```

Comment:
Is it necessary to support this? If not, let's exclude the change in this file. It will generate a wrong input data.

Reply:
Without this change, error is thrown.
```
terminate called after throwing an instance of 'std::runtime_error'
  what():  NYI data type.
```
as in memory uses `int8_t` I've used `H5::PredType::NATIVE_INT8;`

Reply:
I guess that this problem will be gone with https://github.com/Samsung/ONE/pull/12743#discussion_r1525742386

Reply:
Hm... I don't have any plan to support `INT8` in tflchef ... 
Creating `INT8` tflite file is one thing but need to support all through luci and others for validation, which is related with this `compiler/common-artifacts/src/TestDataGenerator.cpp`
Maybe I shouldn't do this in  as of now -_-;